{"name": "test_toggle_password_visibility[firefox]", "status": "broken", "statusDetails": {"message": "TypeError: WebDriver.__init__() got an unexpected keyword argument 'executable_path'", "trace": "request = <SubRequest 'setup' for <Function test_navigate_to_recovery_page[firefox]>>\n\n    @pytest.fixture(params=[\"chrome\", \"firefox\"], scope=\"class\")\n    def setup(request):\n>       driver = initialize_driver(request.param)\n\nconftest.py:6: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nbrowser_name = 'firefox'\n\n    def initialize_driver(browser_name):\n        if browser_name == \"chrome\":\n            driver = webdriver.Chrome(ChromeDriverManager().install())\n        elif browser_name == \"firefox\":\n>           driver = webdriver.Firefox(executable_path=GeckoDriverManager().install())\nE           TypeError: WebDriver.__init__() got an unexpected keyword argument 'executable_path'\n\nutils\\driver_setup.py:9: TypeError"}, "parameters": [{"name": "setup", "value": "'firefox'"}], "start": 1730625016451, "stop": 1730625016451, "uuid": "72c4f30f-f430-4dfc-abb6-a9ffaa45dad3", "historyId": "7cd7f119dbfa56e8231bcd2c43b3776b", "testCaseId": "394b41bf0fcdaf89377fee35a458d11f", "fullName": "tests.test_password_recovery.TestPasswordRecovery#test_toggle_password_visibility", "labels": [{"name": "tag", "value": "@pytest.mark.usefixtures('setup')"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_password_recovery"}, {"name": "subSuite", "value": "TestPasswordRecovery"}, {"name": "host", "value": "WorkPC"}, {"name": "thread", "value": "9892-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_password_recovery"}]}